test/service/test_manifest_async.py

```diff
@@ -179,18 +206,18 @@ def test(self, mock_uuid, mock_helper):
             helper.add_passthru(self.base_url)
             for fetch in (True, False):
                 with self.subTest(fetch=fetch):
-                    manifest_url = 'https://url.to.manifest'
+                    manifest_url = furl(self.base_url)
                     execution_id = '6c9dfa3f-e92e-11e8-9764-ada973595c11'
                     mock_uuid.return_value = execution_id
                     format_ = ManifestFormat.compact
                     filters = {'organ': {'is': ['lymph node']}}
                     params = {
-                        'format': format_.value,
+                        'filters': json.dumps(filters),
                         'catalog': self.catalog,
-                        'filters': json.dumps(filters)
+                        'format': format_.value
```

Comment:
This change provides no improvement, functional or aesthetic.

Reply:
(as far as I can tell. If something subtle is going on with the key order, needs a comment).